src/main/kotlin/com/cycode/plugin/components/toolWindow/components/scanContentTab/ScanContentTab.kt

```diff
@@ -12,54 +12,40 @@ import javax.swing.JPanel
 
 class ScanContentTab : Component<CycodeService>() {
     override fun getContent(service: CycodeService): JPanel {
-        return JPanel().apply {
-            layout = GridBagLayout()
-            add(add(JPanel().apply {
-                add(createClickableLabel(CycodeBundle.message("scanTabTitleLabel")))
-            }), GridBagConstraints().apply {
-                gridy = 0
-                insets = JBUI.insetsBottom(10)
-                anchor = GridBagConstraints.NORTHWEST
-            })
-            add(add(JPanel().apply {
-                add(createClickableLabel(CycodeBundle.message("scanTabPreButtonsLabel")))
-            }), GridBagConstraints().apply {
-                gridy = 1
-                insets = JBUI.insetsBottom(10)
-                anchor = GridBagConstraints.NORTHWEST
-            })
-            add(JButton(CycodeBundle.message("scanTabSecretsBtn")).apply {
-                addActionListener {
-                    service.startSecretScanForCurrentFile()
-                }
-            }, GridBagConstraints().apply {
-                gridy = 2
-                insets = JBUI.insetsBottom(10)
-                fill = GridBagConstraints.HORIZONTAL
-            })
-            add(JButton(CycodeBundle.message("scanTabScaBtn")).apply {
-                addActionListener {
-                    service.startScaScanForCurrentProject()
-                }
-            }, GridBagConstraints().apply {
-                gridy = 3
-                insets = JBUI.insetsBottom(10)
-                fill = GridBagConstraints.HORIZONTAL
-            })
-            add(add(JPanel().apply {
-                add(createClickableLabel(CycodeBundle.message("scanTabOnSaveTip")))
-            }), GridBagConstraints().apply {
-                gridy = 4
-                insets = JBUI.insetsBottom(10)
-                anchor = GridBagConstraints.NORTHWEST
-            })
-            add(add(JPanel().apply {
-                add(createClickableLabel(CycodeBundle.message("howToUseLabel")))
-            }), GridBagConstraints().apply {
-                gridy = 5
-                insets = JBUI.insetsBottom(10)
-                anchor = GridBagConstraints.NORTHWEST
-            })
+        val gbc = GridBagConstraints().apply {
+            gridy = 0
+            insets = JBUI.insetsBottom(10)
+            anchor = GridBagConstraints.NORTHWEST
+            fill = GridBagConstraints.HORIZONTAL
         }
+
+        val panel = JPanel(GridBagLayout())
+
+        gbc.gridy++
```

Comment:
I don't like the idea of increasing this param after each `.add`. I suggest to keep each component in a single method and put `++` logic inside the methods. Or have some static `AddComponent` method that will do this logic

Reply:
sure